pkg/allocator/allocator.go

```diff
@@ -595,7 +594,7 @@ func (a *Allocator) lockedAllocate(ctx context.Context, key AllocatorKey) (idpoo
 		return 0, false, false, fmt.Errorf("no more available IDs in configured space")
 	}
 
-	kvstore.Trace("Selected available key ID", nil, logrus.Fields{fieldID: id})
+	kvstore.Trace(a.logger, "Selected available key ID", slog.Any(fieldID, id))
```

Comment:
Nit. Any reasons for using `slog.Any` here? Same below.

Reply:
ah. I was expecting to the linter to catch these and I forgot to clean them up before submitting the PR. Thanks for double checking it!